framework/include/outputs/ConsoleUtils.h

```diff
@@ -62,12 +62,12 @@ std::string outputNonlinearSystemInformation(FEProblemBase & problem);
 /**
  * Output action RelationshipManager information
  */
-std::string outputRelationshipManagerInformation(MooseApp & app);
+std::string outputRelationshipManagerInformation(const MooseApp & app);
 
 /**
  * Output execution information
  */
-std::string outputExecutionInformation(MooseApp & app, FEProblemBase & problem);
+std::string outputExecutionInformation(const MooseApp & app, FEProblemBase & problem);
```

Comment:
And here - if you feel in the mood - but not a big deal

Reply:
Not super in the mood. I'd be a lot happier if all of these interface where able to use `const` interfaces but they can't. They are just free methods in a util class that nobody but the Console calls.